src/providers/dynamic-links/dynamic-links.ts

```diff
@@ -26,6 +30,36 @@ export class DynamicLinksProvider {
       ? await this.onDynamicLink()
       : await this.getDynamicLink();
     this.logger.debug('Firebase Dynamic Link Data: ', JSON.stringify(dynLink));
+
+    if (typeof dynLink === 'string') {
+      this.logger.debug('Universal Link');
+
+      // defer flag is coming from FCM - marks only "last" links
+      const isDeferred = dynLink.includes('defer');
+      const timeout = isDeferred ? 2000 : 0;
+
+      if (isDeferred) {
+        this.onGoingProcessProvider.set('generalAwaiting');
+        dynLink = dynLink.split('?defer')[0];
+      }
+
+      const subscription = this.platformProvider.platformReady$.subscribe(
+        (ready: boolean) => {
+          if (ready) {
+            setTimeout(() => {
+              this.incomingDataProvider.redir(decodeURIComponent(dynLink), {
+                force: true
```

Comment:
Getting this error:
```
ERROR in src/providers/dynamic-links/dynamic-links.ts(51,17): error TS2345: Argument of type '{ force: boolean; }' is not assignable to parameter of type 'RedirParams'.
  Object literal may only specify known properties, and 'force' does not exist in type 'RedirParams'.
```

Reply:
Had it added in the wallet connect PR. Went ahead and added it to this one